cmd/aries-agent-mobile/main_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ariesagent
+
+import (
+	"testing"
+
+	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers/command"
+	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers/rest"
```

Comment:
move these lints under testify/require import with and remove //nolint below

Reply:
don't resolve this comment, it should be fixed.. you should move Aries-framework-go imports as the last group of imports. This is a general rule, local imports as the last group of imports, preceded by (a blank line and) other libraries imports, preceded by (blank line and) core Go imports. Always follow this rule, otherwise you will get linter errors.

replace these imports as follows:
```
 	"testing"

 	"github.com/stretchr/testify/require"

 	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers/command"
 	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers/rest"
```

never use //nolint on imports